operator/templates/kafka-connect.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       containers:
       - name: kafka-connect
         image: mesosphere/kafka:2.5.1-1.3.1
-        imagePullPolicy: Always
+        imagePullPolicy: {{ .Params.KAFKA_IMAGE_PULL_POLICY }}
```

Comment:
this yaml is named kafka connect, how come we're not using the connect policy?

Reply:
Because it's also using the standard kafka image. The pull policy is tied to the image, not to the features of the operator. Otherwise, I feel that the pull policies would become too fine grained.